src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2859,4 +2940,7 @@ abbrev zeroExtend_truncate_succ_eq_zeroExtend_truncate_or_twoPow_of_getLsbD_true
 @[deprecated and_one_eq_setWidth_ofBool_getLsbD (since := "2024-09-18")]
 abbrev and_one_eq_zeroExtend_ofBool_getLsbD := @and_one_eq_setWidth_ofBool_getLsbD
 
+@[deprecated sshiftRight_msb_eq_msb (since := "2024-10-03")]
+abbrev msb_sshiftRight := @sshiftRight_msb_eq_msb
+
```

Comment:
You want to rename the implementation of `sshiftRight_msb_eq_msb` to `msb_sshiftRight` and then express `sshiftRight_msb_eq_msb` as `msb_sshiftRight`.